app/src/main/res/layout/fragment_skill_details.xml

```diff
@@ -33,6 +33,9 @@
                     android:id="@+id/skill_detail_title"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:text="@string/skill_name"
+                    android:textSize="20sp"
+                    android:layout_marginTop="4dp"
                     android:layout_marginTop="8dp"
                     android:background="@color/md_white_1000"
                     android:text="@string/skill_name"
```

Comment:
there is android:text="@string/skill_name" already present but it is added again